anomalib/models/cflow/config.yaml

```diff
@@ -1,74 +1,79 @@
 ckpt_path: null
-dataset:
-  name: mvtec #options: [mvtec, btech, folder]
-  format: mvtec
-  path: ./datasets/MVTec
-  category: bottle
-  task: segmentation
-  image_size: 256
-  train_batch_size: 16
-  test_batch_size: 16
-  inference_batch_size: 16
-  num_workers: 8
-  transform_config:
-    train: null
-    val: null
-  create_validation_set: false
+seed_everything: null
+results_dir:
+  path: ./results
+  unique: false
+
+data:
+  class_path: anomalib.data.MVTec
+  init_args:
+    root: ./datasets/MVTec
+    category: bottle
+    image_size: [256, 256]
+    train_batch_size: 16
+    test_batch_size: 16
+    num_workers: 8
+    task: segmentation
+    transform_config_train: null
+    transform_config_val: null
+    seed: 0
+    create_validation_set: false
 
 model:
-  name: cflow
-  feature_extractor:
-    backbone: wide_resnet50_2
-    pre_trained: true
-    layers:
-      - layer2
-      - layer3
-      - layer4
-  decoder: freia-cflow
-  condition_vector: 128
-  coupling_blocks: 8
-  clamp_alpha: 1.9
-  fiber_batch_size: 64
-  permute_soft: false
-  lr: 0.0001
-  early_stopping:
-    patience: 2
-    metric: pixel_AUROC
-    mode: max
-  normalization_method: min_max # options: [null, min_max, cdf]
+  class_path: anomalib.models.Cflow
+  init_args:
+    feature_extractor:
+      backbone: wide_resnet50_2
+      pre_trained: true
+      layers:
+        - layer2
+        - layer3
+        - layer4
+    fiber_batch_size: 64
+    decoder: freia-cflow
+    condition_vector: 128
+    coupling_blocks: 8
+    clamp_alpha: 1.9
+    permute_soft: false
+
+optimizer:
+  class_path: torch.optim.Adam
+  init_args:
+    lr: 0.0001
+
+post_processing:
+  normalization_method: MIN_MAX # <null, min_max, cdf>
+  threshold_method: ADAPTIVE # options: [adaptive, manual]
+  manual_image_threshold: null
+  manual_pixel_threshold: null
 
 metrics:
-  image:
+  image_metrics:
```

Comment:
Similar to other class_paths, should we define the metrics with full class_paths such as `torchmetrics.F1Score etc?`

Reply:
The problem is that it might become too verbose 
![image](https://user-images.githubusercontent.com/17232914/210581891-be8b2755-2ede-47d6-80be-ee7f055b01f1.png)
But we do support adding metrics like this since PR https://github.com/openvinotoolkit/anomalib/pull/688

Reply:
That's fair. I just wanted to know if users were aware of this support.